autoload/minimap.py

```diff
@@ -162,20 +164,33 @@ def draw(lengths,indents, startline=0):
             highlight_group = vim.eval("g:minimap_highlight")
             lengths = []
             indents = []
-
-            first = max(1, topline - 80)
-            last = min(bottomline + 80, len(src.buffer))
-            for line in range(first, last):
-                linestring = src.buffer[line]
-                indents.append(len(linestring) - len(linestring.lstrip()))
-                lengths.append(len(linestring))
-
             vim.command(":setlocal modifiable")
 
-            minimap.buffer[:] = draw(lengths,indents)
+            first = max(1, topline - 160)
+            last = min(bottomline + 160, len(src.buffer))
+            scale = 4
+
+            # TODO: this check could be done once at the start
+            #text_minimap_path = shutil.which("text-minimap")
+            #if text_minimap_path is None:
```

Comment:
Oh. Thanks for helping me with this. That would explain it... 

¯\_(ツ)_/¯ Python...

Anyway, I've been using this plugin with my changes and there are some weird things going on with selection on the original source code buffer, and I'm not sure if it's because of my changes. Need to investigate. Just letting you know.

Though I'm really happy about the speed now. :)